pkg/apis/v1/labels.go

```diff
@@ -151,6 +157,7 @@ var (
 	LabelInstanceAcceleratorManufacturer      = apis.Group + "/instance-accelerator-manufacturer"
 	LabelInstanceAcceleratorCount             = apis.Group + "/instance-accelerator-count"
 	LabelNodeClass                            = apis.Group + "/ec2nodeclass"
+	LabelTenancy                              = apis.Group + "/tenancy"
```

Comment:
Discussed a bit offline with other maintainers, I think we should name this `instance-tenancy` rather than `tenancy`. This aligns better with the other label names and it is a property of the instance so I think it's appropriate here. Let us know what you think.